generators/server/templates/src/main/java/package/security/KeycloakEndSessionEndpoint.java.ejs

```diff
@@ -14,36 +19,59 @@ import javax.annotation.Nullable;
 import javax.inject.Named;
 import java.net.URI;
 
+import java.util.HashMap;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+
 @Named("oidc")
-public class KeycloakEndSessionEndpoint implements EndSessionEndpoint {
+public class KeycloakEndSessionEndpoint extends AbstractEndSessionRequest {
```

Comment:
Should this class be renamed since it's not Keycloak-specific?

Reply:
:) Yes I updated my comment already.

Reply:
Renamed it. We should mention that in release/upgrade notice to delete the single file. Hopefully we come up with a generic solution in the vue blueprint, but for now we are fine with manual actions by the users.